### PR TITLE
feat(categorize, hotspot, image-cloze-association): extent fontSizeFa…

### PR DESCRIPTION
**Pull request by arimieandreea**
…ctor to new elements PD-4389

### CODE REVIEW

**Comment by arimieandreea** [comment]
https://illuminate.atlassian.net/browse/PD-4389

File: packages/image-cloze-association/src/root.jsx [hunk 1]
```diff
@@ -22,9 +22,6 @@ const styles = (theme) => ({
     color: color.text(),
     backgroundColor: color.background(),
   },
-  stimulus: {
-    fontSize: theme.typography.fontSize,
-  },
```
Comment (PatriciaRomaniuc) [inline packages/image-cloze-association/src/root.jsx:27]: I think classes.stimulus is still used in this file

**Comment by PatriciaRomaniuc** [comment]
Also, I think the fontSizeFactor should also be added to the documentation of each item type that it's used in

File: packages/categorize/src/categorize/choice.jsx [hunk 2]
```diff
@@ -63,7 +63,6 @@ const styles = (theme) => ({
   cardRoot: {
     color: color.text(),
     backgroundColor: color.white(),
-    fontSize: theme.typography.fontSize + 2,
```
[the remark this answers is not in the document]
Comment (arimieandreea) [reply packages/categorize/src/categorize/choice.jsx:66]: FontSizeFactor does not give the value of fontSize property anyway. It is used for computing it together with root fontsize. (browser fontsize, body fontsize in ibx). If we delete it the fontSize of the cardRoot will inherit it from the parent as all components that we use and do not set fontsize for them

[the remark this answers is not in the document]
Comment (PatriciaRomaniuc) [reply packages/categorize/src/categorize/choice.jsx:66]: so it does not matter that here it was `+2` ?

[the remark this answers is not in the document]
Comment (arimieandreea) [reply packages/categorize/src/categorize/choice.jsx:66]: When we start implementing this https://illuminate.atlassian.net/browse/PD-4388 there were some corner cases like this discussed with George and Andrei Miron as Show Correct Answer or Feedback etc. As I understood from this comments and discussions it is ok that the fontSize to be inherited. (most cases it is usually rootfontsize 16px (browser font size) for these elements). The same problem was for tables (where this was wanted) and this is the reason why I follow the same solution as long as no style was broken during my tests here. If you find a downside or something is weird we can discuss and see a better solution. Thank you

[the remark this answers is not in the document]
Comment (PatriciaRomaniuc) [reply packages/categorize/src/categorize/choice.jsx:66]: ok, so the short answer is that +2 does not matter in this case

**Pull request merged**